app/util/k8s/dcapt-snapshots.json

```diff
@@ -358,6 +358,36 @@
           }
         ],
         "build_number": "8804"
+      },
+      {
+        "version": "8804",
```

Comment:
@SergeyMoroz0703 does not look correct ^^

Reply:
Yep, will decline it.